test/elixir/test/changes_async_test.exs

```diff
@@ -34,12 +34,11 @@ defmodule ChangesAsyncTest do
       )
 
     changes = process_response(req_id.id, &parse_chunk/1)
-    {changes_length, last_seq_prefix} = parse_changes_response(changes)
-    assert changes_length == 1, "db should not be empty"
-    assert last_seq_prefix == "1-", "seq must start with 1-"
```

Comment:
It is the first time I've seen a 4.x change seq. It is completely different from the previous format. I see that we can not check anything about the seq.

Reply:
The format of the `seq`s is different now. They no longer have the predictability they had in previous releases.